site/en/blog/strapi-custom-api.md

```diff
@@ -151,21 +151,15 @@ In this step, we use the service to create a function to get the reading list an
 `api/reading-list-search/services/reading-list-search.js`:
 
 ```ts
+import { errors } from '@strapi/utils';
+
+const { ApplicationError } = errors;
+
 async function search(keyword?: string) {
   try {
-    const entries = await strapi.entityService.findMany('api::reading-list.reading-list', {});
-
-    let filteredEntries = entries;
-    if (keyword) {
-      filteredEntries = entries.filter((entry) => {
-        const keywords = entry.keyword.split('\n').map((keyword) => keyword.trim());
-        return keywords.includes(keyword);
-      });
-    }
-
-    return filteredEntries;
+    return await strapi.entityService.findMany('api::reading-list.reading-list', { filters: { keyword: { $eq: keyword }}});
```

Comment:
All items are correct, thank you

Only this line can't search in a list, I divide the keywords based on the new line, can handle it in other way?